src/components/SuperDrawer/useSuperDrawer.test.tsx

```diff
@@ -133,6 +133,22 @@ describe("useSuperDrawer", () => {
     // Then expect the drawer to not close.
     expect(hook.closeDrawer()).toBeFalsy();
   });
+
+  it("calls onClose when closed", () => {
+    const onClose = jest.fn();
+
+    // Given the useSuperDrawer hook
+    const hook = renderHook(useSuperDrawer, { wrapper }).result.current;
```

Comment:
**Praise:** Really digging the renderHook utility!

Reply:
This is actually my first time using it / even realizing that we used it. :-) Does seem pretty neat

Reply:
I saw it in a FE interview submission and gave it a go!